storage/kv_store_rocksdb_int_keys.py

```diff
@@ -40,3 +40,12 @@ def get_equal_or_prev(self, key):
         except StopIteration:
             value = None
         return value
+
+    def get_last_key(self):
```

Comment:
Please add a test to `test_kv_storages`

Reply:
already added into storage/test/test_ts_store.py